aliroot.sh

```diff
@@ -8,6 +8,7 @@ requires:
   - GEANT3
   - GEANT4_VMC
   - Vc
+  - AliTPCCommon
```

Comment:
So, as we've discussed, `AliTPCCommon` should be visible at build time only to AliRoot and O2. Hence, it should be under the `build_requires:` stanza.